docs/pages/docs/connectors/gnosis.mdx

```diff
@@ -0,0 +1,44 @@
+# GnosisConnector (Gnosis, etc.)
```

Comment:
Could this be changed to `Safe`? The Safe project has spin out of gnosis recently. Also, there are other Gnosis' projects such as Gnosis Chain, etc